apps/cloudfront-functions/src/viewer-request-handler.ts

```diff
@@ -15,9 +15,15 @@ const handler = (
 
     // Check if the uri refers to a gitbook assets
     const isGitbookAssets = uri.startsWith('/gitbook/docs');
+    const isWoff2 = uri.endsWith('.woff2'); // woff2 is a font format
```

Comment:
If you change the regex, this should no longer be needed

Reply:
If I change the regex, this test case breaks:

```js
const example1 = '/i-s/g/mo/v1.0';
expect(handler(makeEvent(example1)).uri).toBe(`${example1}.html`);
```

Reply:
Great, then it's better to specify the extension as you've already done